### ISSUE TITLE
Table for the junction types

### ISSUE THREAD

**Issue by uzluisf** [LaurentRosenfeld/think_raku#1]
This is what I ended up with ([Latex source](https://www.overleaf.com/read/gknwcpksxstf)) after tweaking [an example from SO](https://tex.stackexchange.com/questions/112343/beautiful-table-samples/112382#112382) for the [junction types](https://github.com/LaurentRosenfeld/think_raku/blob/master/book/data_struct_selection.tex#L235-L243).

**Comment by LaurentRosenfeld** [comment]
Thank you very much, Luis. 

I think I know the stupid error that I did: I only gave three parameters to the \begin{tabular}{llcl}, instead or four (well, in fact, I started the table with three columns and then changed my mind to four columns, but forgot to update the header. To me, this error shows a bug in LaTeX: it should not go into an infinite loop or something similar for such a mistake, but report that there is something wrong on line xxx.

Anyway, thanks again,
Laurent.
